js/react.md

```diff
@@ -101,58 +105,34 @@ The signUpFields array in turn consist of an array of objects, each describing a
 
 {% include sign-up-fields.html %}
 
-The following example will replace all the default sign up fields with the ones defined in the `signUpFields` array.
-```js
-import React, { Component } from 'react';
-import { withAuthenticator } from 'aws-amplify-react';
-
-class App extends Component {
-}
+A Sample signUpFields attribute would look like the following:
 
+```js
 const signUpConfig = {
   header: 'My Customized Sign Up',
   hideAllDefaults: true,
   defaultCountryCode: '1',
   signUpFields: [
     {
-      label: 'Email',
+      label: 'My custom email label',
       key: 'email',
       required: true,
       displayOrder: 1,
       type: 'string'
     },
```

Comment:
I think we should just show a sample for `signUpFields` here rather than huge sample which would require additional configuration to execute without error.

Reply:
I believe that is fine, part of it was to show all of the different options in regards to what can be passed in. Not sure if we want to add another additional callout of what else is available.